SoObjects/Mailer/SOGoMailBaseObject.m

```diff
@@ -206,6 +220,22 @@ - (NGImap4Connection *) imap4Connection
       [imap4 retain];
     }
 
+   // Connection broken, try to reconnect
+   if (![[[imap4 client] isConnected] boolValue])
```

Comment:
Ok I see the problem now, I'll redo the patch to ensure exception is launched in nil case also